simple_examples.py

```diff
@@ -177,4 +195,18 @@ def test_header(self):
                     header.dict()
                 )
 
+            def test_pose_job(self):
+                # incoming data from some external source
+                incoming_data = {
+                    'assessment_type': 'GAIT'
+                }
+
+                job = PoseJob(**incoming_data)
```

Comment:
I honestly didn't expect this to work. I thought we'd have a nightmare of a time converting text strings into enums, but this is perfect. We just need to remember to use `AssessmentType.GAIT.value` whenever we want to use the actual value, and not the functional constant.

Reply:
^ this will have an effect on how we store data. We may need to have utility methods on the classes to grab the `value` from a stored enum.

Ex. calling `job.dict()` above produces the following:
```python
{'assessment_type': <AssessmentType.GAIT: 'GAIT'>}
```
Note that the value of `assessment_type` is the functional constant, and not the value that we would want to store.

Reply:
Yeah, that is one of the caveats with python enums. However I have a fix using `(str, Enum)`
```python
class TextAlign(str, Enum):
    LEFT = 'LEFT'
    RIGHT = 'RIGHT'
    CENTER = 'CENTER'

TextAlign.LEFT == TextAlign.LEFT.value and TextAlign.LEFT == 'LEFT'
```

Reply:
p.s. I'm assuming `pydantic` is handling it internally.

Reply:
Yeah, using enums can be considered a bit of an edge case. In the example, I’m using it specifically as a way to input a value into a function (like in OpenCV, `cv2.cvtColor(img, cv2.COLOR_GRAY2BGR)`, that kind of thing). In your example your using it in combination with `pydantic` to have an input from outside python (i.e. json / javascript) that has to be a string, which is a great use case to add. I would consider it an edgecase.

Outside of these two examples, I can't think of any other use valid use cases. In the event that there was a restriction on both the non python input and we couldn't use `pydantic`, then just having an int map exactly like OpenCV seems like the most sensible option, or we would have to map the input to the Enum. Either way its an extreme edgecase.